polygeist/lib/Dialect/Polygeist/Transforms/SYCLRaiseHost.cpp

```diff
@@ -1752,6 +2017,14 @@ void SYCLRaiseHostConstructsPass::runOnOperation() {
 
   if (failed(applyPatternsAndFoldGreedily(scopeOp, frozen)))
     signalPassFailure();
+
+  rewritePatterns.clear();
+  rewritePatterns.add<RaiseScheduleKernel>(context);
+
+  FrozenRewritePatternSet frozen2(std::move(rewritePatterns));
+
+  if (failed(applyPatternsAndFoldGreedily(scopeOp, frozen2)))
+    signalPassFailure();
```

Comment:
`rewritePatterns` was moved here. Can we create another `RewritePatternSet` instead? Alternatively, add to current one, with `benefit=1` and increase all other patterns benefit `+1`

Reply:
Went with the alternative, much nicer, thanks.